libraries/mithril/meta/summary.md

```diff
@@ -0,0 +1,10 @@
+<h4 id="mithril-handling-data">Handling data</h4>
+
+Mithril will pass data to an element as properties, as long as the property
```

Comment:
Based on the conversation in https://github.com/MithrilJS/mithril.js/issues/2220 it seems like this isn't true yet for arrays and objects?

Reply:
Ah, very good point. I've fixed it, and I'll make a new pull request once https://github.com/MithrilJS/mithril.js/pull/2221 lands.